python/ray/data/_internal/execution/operators/actor_pool_map_operator.py

```diff
@@ -290,8 +290,9 @@ def base_resource_usage(self) -> ExecutionResources:
         )
 
     def current_processor_usage(self) -> ExecutionResources:
-        # Both pending and running actors count towards our current resource usage.
-        num_active_workers = self._actor_pool.current_size()
+        # Only count running actors towards our current resource usage to avoid
+        # the progress bar from displaying more GPUs than actually used.
```

Comment:
could we add a comment here, explicitly noting we exclude pending actors from the total count?

Reply:
Sure thing! Done.